.github/workflows/release.yml

```diff
@@ -0,0 +1,117 @@
+name: Publish to crates.io
+
+# on:
```

Comment:
Remove unused/commented out code

Reply:
I plan to use the commented out code instead of trigger on PR, which is only me testing it out.

I think when we cut the tag from github means we ready for release. I would like to automate this process, instead three different manual steps to publish